### DISCUSSION TITLE
Managed Portal: Access contextual data from other widgets

### DISCUSSION THREAD

**Discussion by RickOttenNS** [Azure/api-management-developer-portal#2591] [Feature request]
We would like to access dynamic, contextual data from other, pre-defined widgets in the Managed portal. For example: the user navigates to a specific product. We would then like use the description or ID for said product in one of our own Text Block or custom HTML widgets on the same page, just as we could in the deprecated dev portal. 

Is this possible already? If so: is this documented anywhere? If not: could this be added?

**Comment by ghost** [comment]
@RickOttenNS, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@RickOttenNS, thank you for requesting this feature.

**Comment by ghost** [comment]
@RickOttenNS, thank you for requesting this feature.
